server/server.py

```diff
@@ -60,6 +69,8 @@ async def session(websocket: WebSocket, token):
     try:
         while True:
             msg = await websocket.receive_text()
+            if msg['action'] == 'accept_challenge':
+                accept_challenge(msg)
```

Comment:
If you're going to add more action, it's better to have them parameterized or in some sort a `Factory`

Reply:
it is something that we are going to solve.